apps/ai-image-generator/backend/src/actions/aiig-select-fill.ts

```diff
@@ -44,7 +46,11 @@ export const handler = async (
       throw new Error(`Unable to fetch maskUrl: ${maskUrl}`);
     }
 
-    const { mask: maskBuffer, image: imageBuffer } = await transformImages({
+    const {
+      mask: maskBuffer,
+      image: imageBuffer,
+      sourceStartingDimensions,
```

Comment:
I think this will be cleaned up in the cleanup PR you mentioned but I think there's a good re-design here with these classes where you shouldn't have to get these `sourceStartingDimensions` from a `transformImages` function. I know this isn't the point of the PR and I definitely think this should be part of the refactor. Just wanted to note this as an overall point

Reply:
yeah 👍 . this is a thing i'll try to take a step back and refactor in a follow up :)